### PR TITLE
[4.2] Installation updateuserids

### PR DESCRIPTION
**Pull request by brianteeman**
### Summary of Changes
during the installation we run various queries to update default data with the superuseerid created during the install

### Testing Instructions
Clean install from the generated full package
Check that you never get an error message 
image

### Actual result BEFORE applying this Pull Request
Running queries on tables that have no data 

### Expected result AFTER applying this Pull Request
No change to the generated install

### Documentation Changes Required
none

### CODE REVIEW

**Comment by richard67** [comment]
In 3.10 the removed entries might be needed because one can chose to install sample data at the end of a new installation. But on 4.1 these entries seem to be not needed because the tables are empty after a new installation. So by review this PR seems to be right.

**Review by wilsonge — APPROVED** [review]

File: installation/src/Model/ConfigurationModel.php [hunk 1]
```diff
@@ -295,17 +295,8 @@ protected function updateUserIds($db)
 
 		// Update all core tables created_by fields of the tables with the random user id.
```
Comment (Quy) [inline installation/src/Model/ConfigurationModel.php]:
```suggestion
		// Update core tables created_by fields of the tables with the random user id.
```
Maybe?

**Comment by brianteeman** [comment]
rebased to 4,2 now that there will be no further 4.1 releases

**Comment by wilsonge** [comment]
This is clearly correct given the change in sample data in J4 and given it's not getting tests choosing to merge this on review.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**